### DISCUSSION TITLE
plain-text for Weblate Alert mails

### DISCUSSION THREAD

**Discussion by buhtz** [WeblateOrg/weblate#8528] [General]
Is it possible for the weblate hoster admin to setup weblate that alert mails are plain text only without any HTML in it?

**Comment by nijel** [comment]
It's currently not supported. The notifications are always multipart having both HTML and text parts, your client should choose whatever you prefer.

**Reply by buhtz** [reply]
I assume the multipart is broken because there is no plain text shown. My clients do prefer plain text and would display it if they could find it in the "multipart" mail.

I'll offer a screenshot as soon as possible.

**Reply by nijel** [reply]
Just checked random notification and it is indeed correct multipart:


```mail
--===============8142883007671447419==
Content-Type: multipart/alternative;
 boundary="===============1292159974504892731=="
MIME-Version: 1.0


--===============1292159974504892731==
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 8bit

...text part...
  
[Weblate, the libre continuous localization system.](https://weblate.org/)

Generated on Jan. 12, 2023, 4 a.m..


--===============1292159974504892731==
Content-Type: text/html; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: quoted-printable

<!DOCTYPE html>
```
